easybuild/easyconfigs/g/GROMACS/GROMACS-2023.2-cpeGNU-22.12-hipSYCL-GPU.eb

```diff
@@ -0,0 +1,116 @@
+# Based on recipe from Rasmus Kronberg (CSC) 
+#
+easyblock = 'CMakeMake'
+
+name =          'GROMACS'
+version =       '2023.2'
+versionsuffix = '-hipSYCL-GPU' 
```

Comment:
perhaps this should have a VkFFT tag too since without `-D GMX_GPU_FFT_LIBRARY=` the build system will default to that.

Reply:
Is it really resolved?

Reply:
I have merged two VkFFT based recipes into one. I hope it is now less messy.